build/Linux+BSD/portable/copy-libs

```diff
@@ -135,6 +251,9 @@ building_on_CentOS_6() {
   copyLib libpng12.so.0
   copyLib libudev.so.0
   copyLib liblcms.so.1
+
+  # Needed by Gentoo 2.2:
+  copyLib libpulse.so.0
```

Comment:
@shoogle A gentoo user who didn't have pulse reported that the x86 AppImage wouldn't run.  I actually had this same exact issue on my ARMv7 labtop (which didn't have pulseaudio), which is why I had already added it to my function.  But I'm additing it to your function as well, so x86 works without a pre-installed pulse.  (Note: most Live linux distros will have pulse installed, which might be why you didn't notice it was missing).